scipy/ndimage/morphology.py

```diff
@@ -2075,33 +2108,46 @@ def distance_transform_edt(input, sampling=None, return_distances=True,
 
     In addition to the distance transform, the feature transform can
     be calculated. In this case the index of the closest background
-    element is returned along the first axis of the result.
+    element to each foreground element is returned in a separate array.
 
     Parameters
     ----------
     input : array_like
         Input data to transform. Can be any type but will be converted
         into binary: 1 wherever input equates to True, 0 elsewhere.
-    sampling : float or int, or sequence of same, optional
+    sampling : float, or sequence of float, optional
         Spacing of elements along each dimension. If a sequence, must be of
         length equal to the input rank; if a single number, this is used for
         all axes. If not specified, a grid spacing of unity is implied.
-    return_distances : bool, optional
-        Whether to return distance matrix. At least one of
-        return_distances/return_indices must be True. Default is True.
-    return_indices : bool, optional
-        Whether to return indices matrix. Default is False.
-    distances : ndarray, optional
-        Used for output of distance array, must be of type float64.
-    indices : ndarray, optional
-        Used for output of indices, must be of type int32.
+    return_distances, return_indices : bool, optional
+        These flags can be used to indicate whether the distance transform,
+        the feature transform, or both, must be calculated.
+
+        The `return_distances` default is True, and the
+        `return_indices` default is False.
+    distances : float64 ndarray, optional
+        An output array to store the calculated distance transform, instead of 
+        returning it.
+        
+        This parameter is only used when `return_distances` is True.
+        It must be the same shape as `input`.
+    indices : int32 ndarray, optional
+        An output array to store the calculated feature transform, instead of 
+        returning it.
+        
+        Its shape must be `(input.ndim,) + input.shape`.
 
     Returns
     -------
-    distance_transform_edt : ndarray or list of ndarrays
-        Either distance matrix, index matrix, or a list of the two,
-        depending on `return_x` flags and `distance` and `indices`
-        input parameters.
+    distances : float64 ndarray, optional
```

Comment:
It would be nice to retain the information that returns always has size 1

Reply:
As in, always the same size as the input?